.github/workflows/k8s.yaml

```diff
@@ -27,9 +27,10 @@ jobs:
       - name: Checkout code
         uses: actions/checkout@v2
       - name: Start minikube
-        uses: medyagh/setup-minikube@v0.0.3
+        uses: medyagh/setup-minikube@0ff5b450588f2aee88fd0daba616d04a5582e490
```

Comment:
this needs to be
medyagh/setup-minikube@v0.0.7
or 
medyagh/setup-minikube@latest

here is example:
https://github.com/medyagh/test-minikube-example/pull/7

https://github.com/medyagh/test-minikube-example/runs/4621762363?check_suite_focus=true

Reply:
As expected, doesn't work 😢 
https://github.com/change-metrics/monocle/runs/4623008565?check_suite_focus=true

Reply:
Also removing the k8s-version didn't work :(

Reply:
Seems moving from stable to latest works